src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationClient/System/Windows/Automation/Condition.cs

```diff
@@ -38,7 +38,8 @@ override protected bool ReleaseHandle()
         }
 
         // uiaCondition is one of the Uia condition structs - eg UiaCoreApi.UiaAndOrCondition
-        internal static SafeConditionMemoryHandle AllocateConditionHandle(object uiaCondition)
+        internal static SafeConditionMemoryHandle AllocateConditionHandle<T>(T uiaCondition)
```

Comment:
Using generics here allows to call the generic version of Marshal.StructureToPtr and also to avoid boxing of `uiaCondition` every call to `AllocateConditionHandle` passes a struct for `uiaCondition`.